### PR TITLE
add basic css variables for font-family and font-size

### PR DESCRIPTION
**Pull request by hamilton**
It's not currently possible to change tad's styling in an application-specific way. The right way to enable applications to change basic elements of Tad's css is to thread in CSS variables, using Tad's current stylesheet values as the default. Using variables enables us to also change the implementation of Tad's styling while preserving application-specific choices.

With this in mind, this PR adds two variables to Tad: `--tad-font-family` and `--tad-font-size`.

### CODE REVIEW

**Review by antonycourtney — CHANGES_REQUESTED** [review]
Seems totally reasonable, but how about `--tad-font-family` for consistency with `--tad-font-size`?

File: packages/tadviewer/less/tadviewer.less [hunk 1]
```diff
@@ -5,10 +5,10 @@
 @import "~@blueprintjs/popover2/lib/css/blueprint-popover2.css";
 
 .tad-app-pane {
+  font-family: var(--tad-font, "Helvetica Neue", Helvetica, Arial, sans-serif);
```
Comment (hamilton) [inline packages/tadviewer/less/tadviewer.less:8]:
```suggestion
  font-family: var(--tad-font-family, "Helvetica Neue", Helvetica, Arial, sans-serif);
```

**Comment by hamilton** [comment]
good catch. Made the change

**Review by antonycourtney — APPROVED** [review]
looks good! Thanks for addressing PR feedback.

**Pull request merged**